tests/Dialect/Arith/Conversions/ArithToModArith/arith-to-mod-arith.mlir

```diff
@@ -0,0 +1,55 @@
+// RUN: heir-opt --arith-to-mod-arith --split-input-file %s | FileCheck %s --enable-var-scope
+
+// CHECK-LABEL: @test_lower_add
+// CHECK-SAME: (%[[LHS:.*]]: [[T:.*]], %[[RHS:.*]]: [[T]]) -> [[T]] {
```

Comment:
can you change at least one of these tests to specify exactly what the lowered type should look like? Since arith -> mod_arith is making a nontrivial choice here, it would be good to formalize it in the test.